src/Murder/Core/Extensions/EnumExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Runtime.CompilerServices;
+
+namespace Murder.Core.Extensions;
```

Comment:
nit: probably should go under `Utilities/` (we have a lot of extensions there haha)